commit/plugin.go

```diff
@@ -343,6 +344,7 @@ func (p *Plugin) getPriceRelatedObservations(
 
 		if cciptypes.SeqNum(latestPriceOcrSeqNum) >= prevOutcome.MainOutcome.InflightPriceOcrSequenceNumber {
 			lggr.Infow("previous price report made it through", "ocrSeqNum", latestPriceOcrSeqNum)
+			ctx = context.WithValue(ctx, consts.InvalidateCacheKey, true)
```

Comment:
Can we move this closer to where the ctx is used, e.g have a `var invalidateCache bool` up top, set it to `invalidateCache = true` here and before the `go func()` call:

```golang
ctx = context.WithValue(ctx, consts.InvalidateCacheKey, invalidateCache)
```